Makefile

```diff
@@ -61,7 +61,7 @@ check-dependencies:
 lint: lint-js lint-python lint-html lint-links
 lint-js:
 	@yarn eslint --ext js --report-unused-disable-directives "**/*.{js,ts}"
-	@yarn prettier --check "**/*.{js,ts,mjs,cjs,mts,cts,md,sql,json,yml,html,css}"
+	@yarn prettier --write --check "**/*.{js,ts,mjs,cjs,mts,cts,md,sql,json,yml,html,css}"
```

Comment:
It is not appropriate to add `--write` here. That's what the `format-js` target is for.

Reply:
Thanks. I've removed it.